scipy/optimize/_basinhopping.py

```diff
@@ -325,8 +325,8 @@ def __call__(self, **kwargs):
 
 def basinhopping(func, x0, niter=100, T=1.0, stepsize=0.5,
                  minimizer_kwargs=None, take_step=None, accept_test=None,
-                 callback=None, interval=50, disp=False, niter_success=None,
-                 seed=None):
+                 accept_rate=0.5, callback=None, interval=50, disp=False,
+                 niter_success=None, seed=None):
```

Comment:
new keywords need to appear at the end of the signature for backwards compatibility.